app/javascript/components/recipes/base/selected-ingredient-card.vue

```diff
@@ -64,7 +69,9 @@
 </template>
 
 <script>
+import baseDropdown from '../../base/base-dropdown.vue';
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Expected 1 empty line after import statement not followed by another import  import/newline-after-import